c7n/mu.py

```diff
@@ -63,13 +63,17 @@ class PythonPackageArchive(object):
 
     zip_compression = zipfile.ZIP_DEFLATED
 
-    def __init__(self, *modules):
+    def __init__(self, modules=[], cache_file=None):
```

Comment:
use a tuple () instead of a list for default arg.. default args are persistent across invocations.. so using mutables there is always a red flag. a tuple is better immutable iterable default.

```python
def foo(x=[]):
   x.append(len(x))
   return len(x)

>>> [foo() for n in range(10)]
[1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
```